packages/ui-library/tsconfig.json

```diff
@@ -14,7 +14,16 @@
     "experimentalDecorators": true,
     "skipLibCheck": true,
     "strict": true,
-    "useDefineForClassFields": false
+    "useDefineForClassFields": false,
```

Comment:
I'm a little but unclear on what this plugin does. Should it be included within the scope of this ticket?

Reply:
Not sure where this comes from or what it does thb